rainier-core/src/main/scala/com/stripe/rainier/compute/Gradient.scala

```diff
@@ -1,14 +1,11 @@
 package com.stripe.rainier.compute
 
 import com.stripe.rainier.ir._
-import scala.collection.generic.CanBuildFrom
 import scala.collection.mutable.HashMap
 
 private object Gradient {
 
-  def derive[F[α] <: TraversableOnce[α]](variables: F[Variable], output: Real)(
```

Comment:
Why drop the CanBuildFrom and whatnot?

Reply:
Frankly just because it's stylistically fancier and more general than I think is needed or than is standard in this codebase, and so when I ran across a need to change it, I took the opportunity to get rid of it.